rocky/reports/report_types/aggregate_organisation_report/report_design.html

```diff
@@ -0,0 +1,584 @@
+{% load i18n %}
+{% load static %}
+
+<main id="main-content"
+      tabindex="-1"
+      class="sidemenu choose-report report"
+      lang="nl">
+    {% comment %} {% include "partials/report_sidemenu.html" %} {% endcomment %}
+    {% include "partials/report_sidemenu.html" %}
+
+    <article>
+        <section class="introduction">
+            <div class="horizontal-view toolbar">
+                <button>Save report</button>
+            </div>
+            <div>
+                <h1>Mockup rapportage OpenKAT</h1>
+                <p>Dit is het OpenKAT rapport voor organisatie Organisatienaam.</p>
```

Comment:
Dutch / untranslated?

Reply:
This is the design html, it is not rendered